console/core/src/main/java/org/eclipse/kapua/app/console/core/server/GwtSettingsServiceImpl.java

```diff
@@ -52,8 +52,11 @@ public String getSsoLoginUri() throws GwtKapuaException {
     @Override
     public String getSsoLogoutUri(String ssoIdToken) throws GwtKapuaException {
         try {
-            return SsoLocator.getLocator(this).getService().getLogoutUri(ssoIdToken,
-                    URI.create(SsoHelper.getHomeUri()), UUID.randomUUID().toString());
+            if (SETTINGS.getBoolean(ConsoleSettingKeys.SSO_OPENID_LOGOUT_ENABLED, true)) {
```

Comment:
Can you please change the header to "2020"? I hope I am not too late 😄

Reply:
done!